plugins/azure/loadbalancer/lbPublicIp.js

```diff
@@ -0,0 +1,54 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Load Balancer Has Public IP',
+    category: 'Load Balancer',
+    domain: 'Availability',
+    description: 'Ensures that Azure Load Balancers has Public IP associated.',
```

Comment:
```suggestion
    description: 'Ensures that Azure Load Balancers have public IPs associated.',
```
use either singular or plural statements